docs/guide/syntax.md

```diff
@@ -377,7 +377,7 @@ blockquote {
 
 Just like you would do in markdown, you can use images pointing to a remote or local URL.
 
-For remote assets, the built-in [`vite-plugin-remote-assets`](https://github.com/antfu/vite-plugin-remote-assets) will cache them into the disk at the first run so you can have instant loading even for large images later on.
+For remote assets, the built-in [`vite-plugin-remote-assets`](https://github.com/antfu/vite-plugin-remote-assets) will cache them onto disk at first run so you can have instant loading--even for large images later on.
```

Comment:
will cache them onto **the** disk at first

Reply:
"Cache to disk" would arguably be the most correct if we're playing inside baseball with storage admins. :D